github-pr-reviewer/code_reviewer/github_code.py

```diff
@@ -0,0 +1,28 @@
+from github import Github
+
+from code_reviewer.reviewer import CodeReviewer
+
+class GithubAPI:
+    def __init__(self, access_token):
```

Comment:
Consider adding a docstring to the GithubAPI class to provide a brief explanation of its purpose and functionality.